site-src/concepts/gamma.md

```diff
@@ -164,8 +164,8 @@ important:
     custom timeouts for that consumer's use of the workload). This Route will
     only affect requests from workloads in the same Namespace as the Route.
 
-    For example, this HTTPRoute would cause all clients of the `smiley`
```

Comment:
I think the original language is a bit clearer, but happy to defer to others

Reply:
For me the qualifier ("in the `fast-clients` Namespace") should be close to the thing being qualified ("all clients"). The previous version implied the service "smiley" was in the "in the `fast-clients` Namespace".

But I admin this is subjective. I can put the "in the `fast-clients` Namespace" in parentheses so it will be:

```For example, this HTTPRoute would cause all clients (in the `fast-clients` Namespace) of the `smiley` workload  to have a 100ms timeout:```

Reply:
I do understand the potential confusion but the correction trips me up too. Can we just split the two phrases up?

> configure all clients in the `fast-clients` Namespace to have a 100ms timeout for requests to `smiley` workloads.